### ISSUE TITLE
Update video modal to work with sites other than YouTube

### ISSUE THREAD

**Issue by stevejalim** [mdn/developer-portal#587]
@stevejalim commented on [Wed Oct 30 2019](https://github.com/mdn/kuma-deployer/issues/13)

While most of our videos will be hosted on YouTube, it's possible that some will be embedded from (say) Vimeo.

However, the [code for the modal](https://github.com/mdn/developer-portal/blob/master/src/js/atoms/modal.js) implies that it's only expecting YouTube as a host, so this needs to be expanded to at least include Vimeo

**Comment by stevejalim** [comment]
@atopal @valgrimm Do either of you think this is a p1 (alpha) blocker, or can it be a p2?

**Comment by atopal** [comment]
yeah, until we have defined what P2 means. General backlog really.
